doc/en/getting-started.rst

```diff
@@ -153,6 +153,57 @@ Once you develop multiple tests, you may want to group them into a class. pytest
 
 The first test passed and the second failed. You can easily see the intermediate values in the assertion to help you understand the reason for the failure.
 
+Grouping tests in classes can be beneficial for the following reasons:
+
+ * Test organization
+ * Sharing fixtures for tests only in that particular class
+ * Applying marks at the class level and having them implicitly apply to all tests
+
+Something to be aware of when grouping tests inside classes is that each test does not have the same instance of the class.
```

Comment:
```suggestion
Something to be aware of when grouping tests inside classes is that each test does have a own unique instance of the class.
```

Reply:
Thanks @RonnyPfannschmidt - thoughts on:

'each test has a unique instance of the class'.

a slight middle ground there?

Reply:
@symonk great idea, i just wanted to get away from "not the same" - your suggestion reads better :+1: